### PR TITLE
refactor(docs-infra): fix `strictTemplates` failures in `accessibility` docs example

### PR DESCRIPTION
**Pull request by vlasischar**
… accessibility -> converted to strict

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by vlasischar** [comment]
@googlebot I signed it!

______
Vlasis Charalampous
Laboratory of Knowledge and Intelligent Computing (KIC)-
University of Ioannina | Software Developer

**Comment by bampakoa** [comment]
@vlasischar it seems that your commit does not contain a body. Can you try to add one so that the `lint` rule can pass successfully? 

@gkalpak can you check the errors regarding the `test_docs_*` CI rules?

**Comment by petebacondarwin** [comment]
So one error is that the accessibility e2e test is expecting `016` rather than `16`. See https://github.com/angular/angular/blob/fc709423f2f796aeaccd69384f365d060a6aa05c/aio/content/examples/accessibility/e2e/src/app.e2e-spec.ts#L15

**Comment by petebacondarwin** [comment]
Another is that a bunch of examples have strict mode errors in the TS code. See [test_docs_examples job failure](https://app.circleci.com/pipelines/github/angular/angular/22888/workflows/ea41b7a7-141c-4ee4-b566-89e86cdb7480/jobs/831297).

For example:

```
ERROR in src/app/app.component.ts:10:3 - error TS2564: Property 'color' has no initializer and is not definitely assigned in the constructor.
10   color: string;
     ~~~~~
src/app/highlight.directive.ts:12:12 - error TS2564: Property 'defaultColor' has no initializer and is not definitely assigned in the constructor.
12   @Input() defaultColor: string;
              ~~~~~~~~~~~~
src/app/highlight.directive.ts:16:26 - error TS2564: Property 'highlightColor' has no initializer and is not definitely assigned in the constructor.
16   @Input('appHighlight') highlightColor: string;
                            ~~~~~~~~~~~~~~
src/app/highlight.directive.ts:26:20 - error TS2345: Argument of type 'null' is not assignable to parameter of type 'string'.
26     this.highlight(null);
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @vlasischar 🚀 

It is beyond the scope of this PR to fix all failures in docs examples, but maybe you could fix the ones in the `accessibility` example. There is one more failure there (see https://github.com/angular/angular/pull/39248#issuecomment-708639456).

While you're at it, could you please also change the commit message to better reflect the changes in this PR. For example:

> refactor(docs-infra): fix \`strictTemplates\` failures in \`accessibility\` docs example

##
To give some context for others, this PR has been created during an Angular Athens meetup workshop (on contributing to Angular). For more details on the issue being addressed here, see angularathens/2nd-workshop-angular-contrib#6.

**Comment by vlasischar** [comment]
@gkalpak @petebacondarwin sorry for the mess. It was my first PR. I'll read the comments, you made, carefully and I'll figure out a way to make a proper commit. I'm here to stay :)

File: aio/tools/examples/shared/boilerplate/cli/tsconfig.json [hunk 1]
```diff
@@ -25,7 +24,6 @@
   "angularCompilerOptions": {
     "strictInjectionParameters": true,
     "strictInputAccessModifiers": true,
-    // TODO(gkalpak): Fix the code and enable this (i.e. switch from `fullTemplateTypeCheck` to `strictTemplates`).
-    "fullTemplateTypeCheck": true,// "strictTemplates": true
+    "strictTemplates": true
```
[the remark this answers is not in the document]
Comment (vlasischar) [reply aio/tools/examples/shared/boilerplate/cli/tsconfig.json:27]: @gkalpak I'll start checking all the examples after resolving the accessibility, first 💪

[the remark this answers is not in the document]
Comment (gkalpak) [reply aio/tools/examples/shared/boilerplate/cli/tsconfig.json:27]: I suspect fixing all errors will be overwhelming. But we don't have to fix them all in one PR. It is perfectly fine to fix just one example (or a couple). You can alays fix more in subsequent PRs, if you feel like it :wink:

**Review by gkalpak — APPROVED** [review]
LGTM 🚀
Thx, @vlasischar! Looking forward to the next PR :wink:

Reviewed-for: global-docs-approvers

**Pull request closed without merging**